app/uk/gov/hmrc/fhregistrationfrontend/views/deregistration/deregistration_acknowledgement.scala.html

```diff
@@ -60,7 +60,7 @@ <h2 class="heading-medium">
   <div class="form-group form-actions">
 
     <p>
-      <a href=@uk.gov.hmrc.fhregistrationfrontend.controllers.routes.SignOutController.signout class="button">@Messages("button.finishAndSignOut")</a>
+      <a href=@uk.gov.hmrc.fhregistrationfrontend.controllers.routes.SignOutController.signout class="button" role="button">@Messages("button.finishAndSignOut")</a>
```

Comment:
It exposes the element as a button (instead of a link) to assistive technology, so the visual appearance (of a button) matches that which is provided audibly to the user via the screen-reader. A js event also hooks in to ensure what looks like a button also acts like a button - that it can be activated via the spacebar as well as the enter key.